src/query/test/index.ts

```diff
@@ -99,6 +99,12 @@ t.test('simple graph', async t => {
     ['#a', ['a']], // identifier
     ['#a:v(1)', ['a']], // matches identifier + semver
     ['#a:v(2)', []], // fails to match identifier + semver
+    [
+      ':path("*")',
+      ['my-project', 'a', 'b', 'c', 'd', 'e', 'f', '@x/y'],
+    ], // path selector for all
```

Comment:
This test would only be matching `my-project` since that's the only file type of node present in the used graph.